docs/pages/Command-Line.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: default
+title: Command-Line
+permalink: /command-line/
+---
+
+- _TODO: update "Command-Line" document [#868](https://github.com/ChainSafe/gossamer/issues/868)_
+
+---
+
+## Gossamer Command
+
+The `gossamer` command is the root command for the `gossamer` package (`cmd/gossamer`). The root command starts the node (and initializes the node if the node has not already been initialized). 
+
+The `gossamer` command accepts the following ***local flags*** and ***global flags***:
+
+```
+--verbosity value  Supports levels crit (silent) to trce (trace) (default: "info")
+--name value       Node implementation name
+--chain value       Node implementation id used to load default node configuration
+--config value     TOML configuration file
+--datadir value    Data directory for the node
```

Comment:
--basedir?

Reply:
yes, I can change here but was not focusing on content in this pull request, see #868